examples/spark2/pom.xml

```diff
@@ -78,7 +83,12 @@
     <dependency>
       <groupId>org.apache.httpcomponents</groupId>
       <artifactId>httpclient</artifactId>
-      <version>${httpclient.version}</version>
+      <version>4.3.4</version>
```

Comment:
Please use the unify version

Reply:
@xubo245 I have already use the unify version of http client library.